CMakeLists.txt

```diff
@@ -119,11 +119,28 @@ if(${GFXRECON_ENABLE_RELEASE_ASSERTS})
 endif()
 
 if(MSVC)
+
+    # The host toolchain architecture (i.e. are the compiler and other tools compiled to ARM/Intel 32bit/64bit binaries):
+    message("CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE: " ${CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE})
+    # The platform name (architecture), we want to build our project for:
+    # e.g. passed through the commandline -A option: cmake -G "Visual Studio 17 2022" -A ARM64
+    message("CMAKE_GENERATOR_PLATFORM: " ${CMAKE_GENERATOR_PLATFORM})
```

Comment:
I can kill these informational messages if they are judged noisy. The second one is kind of helpful if the user forgets the `-A` flag.

Reply:
> I can kill these informational messages if they are judged noisy. The second one is kind of helpful if the user forgets the `-A` flag.

I vote for keeping them--I don't think they're too noisy. It appears that by default `message("...")` will output to stderr and highlight the output red when using CMake GUI. Consider using `message(STATUS "...")` to avoid them being output as errors.

Reply:
Great, we can always lose them later.